.github/actions/bitcoin-int-tests/Dockerfile.generic.bitcoin-tests

```diff
@@ -5,7 +5,15 @@ WORKDIR /src/
 COPY . .
 
 WORKDIR /src/testnet/stacks-node
-RUN cargo test --no-run
+
+RUN rustup override set nightly && \
+    rustup component add llvm-tools-preview && \
+    cargo install grcov
+
+ENV RUSTFLAGS="-Zinstrument-coverage" \
+    LLVM_PROFILE_FILE="stacks-blockchain-%p-%m.profraw"
+
+RUN cargo build
```

Comment:
I think this line should remain:

`RUN cargo test --no-run`

Because the test build is different than the normal cargo build outputs. That way, each individual integration test doesn't also have to perform the build.

Reply:
I changed it here because I need a binary to be produced because it's required for codecov to interpret the generated report properly. I could use `cargo test --no-run`, but I'd have to run `cargo build` following it to get the binary. Unless there's a way to generate the binary using `cargo test`?

Reply:
As discussed offline, updated to run `cargo test --no-run` first then `cargo build` to generate the needed stacks-node binary.